server/base/src/main/java/org/apache/accumulo/server/metadata/iterators/TabletMetadataCheckIterator.java

```diff
@@ -98,11 +97,11 @@ public void seek(Range range, Collection<ByteSequence> columnFamilies, boolean i
 
     var colsToRead = check.columnsToRead();
 
-    source.seek(new Range(tabletRow), Set.of(), false);
+    source.seek(new Range(tabletRow), colsToRead.getFamilies(), true);
```

Comment:
When want to read everything for the empty set need to set inclusive to false.  

```suggestion
    source.seek(new Range(tabletRow), colsToRead.getFamilies(), !colsToRead.getFamilies().isEmpty());
```